lib/cocoapods/generator/copy_resources_script.rb

```diff
@@ -150,7 +153,7 @@ def script
 
       XCASSETS_COMPILE = <<EOS
 
-if [[ -n "${WRAPPER_EXTENSION}" ]] && [ "`xcrun --find actool`" ] && [ `find . -name '*.xcassets' | wc -l` -ne 0 ]
+if [[ -n "${WRAPPER_EXTENSION}" ]] && [ "`xcrun --find actool`" ] && [ `echo $XCASSET_FILES | wc -l` -ne 0 ]
```

Comment:
```bash
[ `echo $XCASSET_FILES | wc -l` -ne 0 ]
```

Even if `XCASSET_FILES` would be empty this would not equal 0. I don't see how this check would work.

```bash
$ XCASSET_FILES=""
$ echo $XCASSET_FILES | wc -l
       1
```